x-pack/plugins/ecs_data_quality_dashboard/server/routes/get_index_stats.test.ts

```diff
@@ -56,6 +62,67 @@ describe('getIndexStatsRoute route', () => {
     expect(response.status).toEqual(500);
     expect(response.body).toEqual({ message: errorMessage, status_code: 500 });
   });
+
+  test('requires date range when isILMAvailable is false', async () => {
+    const request = requestMock.create({
+      method: 'get',
+      path: GET_INDEX_STATS,
+      params: {
+        pattern: `auditbeat-*`,
+      },
+      query: {
+        isILMAvailable: false,
+      },
+    });
+
+    const mockIndices = { 'auditbeat-7.15.1-2022.12.06-000001': {} };
+    (fetchStats as jest.Mock).mockResolvedValue({
+      indices: mockIndices,
+    });
+
+    const response = await server.inject(request, requestContextMock.convertContext(context));
+    expect(response.status).toEqual(403);
```

Comment:
Consider returning a `400` instead of `403` because [400 Bad Request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) is generally indicative of a malformed request from the client:

> The HyperText Transfer Protocol (HTTP) 400 Bad Request response status code indicates that the server cannot or will not process the request due to something that is perceived to be a client error (for example, malformed request syntax, invalid request message framing, or deceptive request routing).

vs [403 Forbidden](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403), which has more of an "auth" flavor:

> The HTTP 403 Forbidden response status code indicates that the server understands the request but refuses to authorize it.
>
> This status is similar to [401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401), but for the 403 Forbidden status code, re-authenticating makes no difference. The access is tied to the application logic, such as insufficient rights to a resource.